pip/pip-278.md

```diff
@@ -0,0 +1,131 @@
+# Background knowledge
+
+Apache Pulsar is a distributed messaging system that supports multiple messaging protocols and storage methods. Among them, Pulsar Topic Compaction is a mechanism to clean up duplicate messages in topics to reduce storage space and improve system efficiency.
```

Comment:
It should be a wrong description of the topic compaction. It is not for removing the duplicated messages. The message deduplication was. The topic compaction provides a key-based data retention mechanism that allows you only to keep the most recent message associated with that key.

Reply:
And we should also add the topic compaction use case of the new load balancer. It has improved the topic compaction only to keep the first value of the key.

And we should also explain if the new compaction service will affect the load balancer or not.

Reply:
Ok, I fix description of the topic compaction